openapi/p4pa-classification.openapi.yaml

```diff
@@ -422,6 +422,34 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ClassificationErrorDTO'
+  /payment-notification:
+    post:
+      tags:
+        - payment-notification
+      operationId: createPaymentNotification
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PaymentNotificationDTO"
+        required: true
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/PaymentNotificationNoPIIDTO"
```

Comment:
why should you return a PaymentNotificationNoPIIDTO if you are creating a PaymentNotificationDTO?
let's return the PaymentNotificationDTO after the save